pkg/webhook/policy.go

```diff
@@ -419,9 +420,16 @@ func (h *validationHandler) validateTemplate(ctx context.Context, req *admission
 	return false, nil
 }
 
+func getAnyObject(req *admission.Request) []byte {
```

Comment:
nit: this might be misleading for the cel engine since anyObject is derived else where. 

```suggestion
func getReqObject(req *admission.Request) []byte {
```